content/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -159,38 +159,37 @@ W&B recommends that you use a Terraform module managed by W&B to provision a sto
 
     This policy grants your AWS account full access to the key and also assigns the required permissions to the AWS account hosting the W&B Platform. Keep a record of the KMS Key ARN.
 
-2. Provision the S3 Bucket
+1. Provision the S3 Bucket
 
     Follow these steps to provision the S3 bucket in your AWS account:
 
     1. Create the S3 bucket with a name of your choice. Optionally create a folder which you can configure as sub-path to store all W&B files.
-    2. Enable bucket versioning.
-    3. Enable server side encryption, using the KMS key from the previous step.
-    4. Configure CORS with the following policy:
+    1. Enable server side encryption, using the KMS key from the previous step.
+    1. Configure CORS with the following policy:
 
         ```json
         [
-            {
-                "AllowedHeaders": [
-                    "*"
-                ],
-                "AllowedMethods": [
-                    "GET",
-                    "HEAD",
-                    "PUT"
-                ],
-                "AllowedOrigins": [
-                    "*"
-                ],
-                "ExposeHeaders": [
-                    "ETag"
-                ],
-                "MaxAgeSeconds": 3600
-            }
+          {
+              "AllowedHeaders": [
+                  "*"
+              ],
+              "AllowedMethods": [
+                  "GET",
+                  "HEAD",
+                  "PUT"
+              ],
+              "AllowedOrigins": [
+                  "*"
+              ],
+              "ExposeHeaders": [
+                  "ETag"
+              ],
+              "MaxAgeSeconds": 3000
+          }
         ]
         ```
-
-    5. Grant the required S3 permissions to the AWS account hosting the W&B Platform, which requires these permissions to generate [pre-signed URLs]({{< relref "./presigned-urls.md" >}}) that AI workloads in your cloud infrastructure or user browsers utilize to access the bucket.
+        {{% alert %}}If data in your bucket expires due to a lifecycle policy, you may lose the ability to read the history of some runs.{{% /alert %}}
```

Comment:
Is there any high-level guidance we can provide on this? I know it'll be cloud specific....

Reply:
Yeah it is Cloud specific. I think this is the sort of thing: https://cloud.google.com/storage/docs/lifecycle, https://learn.microsoft.com/en-us/azure/storage/blobs/lifecycle-management-policy-configure?tabs=azure-portal, https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lifecycle-mgmt.html. Considering all three, I think I will change the text here to "object lifecycle management policy", and link to each cloud's docs in the correct tab.